websites/D/Discords.com/presence.ts

```diff
@@ -0,0 +1,175 @@
+      const page = document.querySelector("li.page-item.active")?.textContent,
+      tag = document.querySelector("h1.text-center")?.textContent.replace("Discord Servers", " ");
+      presenceData.details = `Viewing 🔑 ${tag || "Nothing"} keyword`;
+      presenceData.state = `📖 Page ${page}`;
+      presenceData.buttons = [
+        {
+          label: "View Keyword",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname === "/servers/top-100") {
+      const page = document.querySelector("li.page-item.active")?.textContent;
+      presenceData.details = "Viewing 🏅 top-100 servers";
+      presenceData.state = `📖 Page ${page}`;
+      presenceData.buttons = [
+        {
+          label: "View Top-100",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname.includes("/join")) {
+      const server_name_join = document.querySelector("h2.mt-4")?.textContent;
+      presenceData.details = `Joining 🚦 ${server_name_join || "Nothing"} `;
+      presenceData.buttons = [
+        {
+          label: `Join ${server_name_join || "Nothing"}`,
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname.includes("/upvote")) {
+      const server_name_upvote = document.querySelector("h1.JoinUpvote_clickable-server-name__3l6Es")?.textContent;
+      presenceData.details = `Upvoting 🗳️ ${server_name_upvote || "Nothing"} `;
+      presenceData.buttons = [
+        {
+          label: `Upvote ${server_name_upvote || "Nothing"}`,
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname.includes("/servers/")) {
+      const join = document.querySelectorAll("a.btn")[1]?.getAttribute("href"),
+      upvote = document.querySelectorAll("a.btn")[2]?.getAttribute("href"),
+      membersonline = document.querySelectorAll("span.mr-2")[0]?.textContent.replace(`${document.querySelectorAll("span.mr-2")[0]?.textContent}`, `${"👥" + document.querySelectorAll("span.mr-2")[0]?.textContent}`),
+      members = document.querySelectorAll("span.mr-2")[1]?.textContent.replace(`${document.querySelectorAll("span.mr-2")[1]?.textContent}`, `${"🎁" + document.querySelectorAll("span.mr-2")[1]?.textContent}`),
+      boosts = document.querySelectorAll("span.mr-2")[2]?.textContent.replace(`${document.querySelectorAll("span.mr-2")[2]?.textContent}`, `${"💎" + document.querySelectorAll("span.mr-2")[2]?.textContent}`),
+      server = document.querySelector("h1.servernameh1")?.textContent.replace("PREMIUM", " ");
+      presenceData.details = `Viewing 🎨 ${server}`;
+      presenceData.state = `${membersonline || "N/A"}, ${members || "N/A"}, ${boosts || "No boosts"}`;
+      presenceData.buttons = [
+        {
+          label: "Join",
+          url: `https://discords.com${join}`
+        },
+        {
+          label: "Upvote",
+          url: `https://discords.com${upvote}`
+        }
```

Comment:
No no no, you can do button to redirect to server subpage on this site but not these.

Reply:
I've was a bit iffy with this one as the link does not go straight to a discord link 
<img src="https://cdn.anaxes.xyz/kovg2q4l09a"/>

However, if you want me to change it, I'll happily do so.

Reply:
![image](https://user-images.githubusercontent.com/9348108/118813084-140b3b00-b8af-11eb-8a47-7877edba4e28.png)

Reply:
Yeah well still would be nice to remove them :/

Reply:
Alright, no problem 😃